src/renderer/styles/notification-comp.less

```diff
@@ -62,13 +62,18 @@ body {
 
 .title {
   font-family: sans-serif;
-  font-size: 14px;
   font-weight: 700;
-  color: var(--text-color);
+  width: 100%;
+  overflow-wrap: break-word;
+  font-size: 14px;
+  margin-top: 5px;
```

Comment:
Do we a margin top here? Can you please attach a screenshot of the new notification?

Reply:
probably best to remove that line :)